bundles/org.openhab.binding.kodi/src/main/java/org/openhab/binding/kodi/internal/protocol/KodiConnection.java

```diff
@@ -1214,7 +1214,8 @@ public int getPVRChannelGroupId(final String channelType, final String pvrChanne
 
     public int getPVRChannelId(final int pvrChannelGroupId, final String pvrChannelName) {
         for (KodiPVRChannel pvrChannel : getPVRChannels(pvrChannelGroupId)) {
-            if (StringUtils.equalsIgnoreCase(pvrChannel.getLabel(), pvrChannelName)) {
+            String label = pvrChannel.getLabel();
+            if (label != null && label.equals(pvrChannelName)) {
```

Comment:
Are you sure it is safe to not ignore case?

Reply:
Yes, I am. I thought about it twice.